reference/sql/statements/create-sequence.md

```diff
@@ -0,0 +1,322 @@
+    SELECT next value for seq;
+    ```
+
+    ```
+    +--------------------+
+    | next value for seq |
+    +--------------------+
+    |                 11 |
+    +--------------------+
+    1 row in set (0.00 sec)
+    ```
+
++ Create a sequence object with default custom parameters:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    CREATE SEQUENCE seq2 start 3 increment 2 minvalue 1 maxvalue 10 cache 3;
+    ```
+
+    ```
+    Query OK, 0 rows affected (0.01 sec)
+    ```
+
++ When the sequence object has not been used in this session, the `lastval()` function returns a `NULL` value.
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT lastval(seq2);
+    ```
+
+    ```
+    +---------------+
+    | lastval(seq2) |
+    +---------------+
+    |          NULL |
+    +---------------+
+    1 row in set (0.01 sec)
+    ```
+
++ The first valid value of the `nextval()` function for the sequence object is the starting value.
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT nextval(seq2);
+    ```
+
+    ```
+    +---------------+
+    | nextval(seq2) |
+    +---------------+
+    |             3 |
+    +---------------+
+    1 row in set (0.00 sec)
+    ```
+
++ Although the `setval()` function can change the position of the current value of the sequence object, it cannot change the fixed increment or decrement of the next value.
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT setval(seq2, 6);
+    ```
+
+    ```
+    +-----------------+
+    | setval(seq2, 6) |
+    +-----------------+
+    |               6 |
+    +-----------------+
+    1 row in set (0.00 sec)
+    ```
+
++ When you use `nextval()` to get the next value, the next value will follow the fixed increment or decrement of the sequence.
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT next value for seq2;
+    ```
+
+    ```
+    +---------------------+
+    | next value for seq2 |
+    +---------------------+
+    |                   7 |
+    +---------------------+
+    1 row in set (0.00 sec)
+    ```
+
++ In the following example, you can use the next value of the sequence as the default value for the column.
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    CRATE table t(a int default next value for seq2);
+    ```
+
+    ```
+    Query OK, 0 rows affected (0.02 sec)
+    ```
+
++ In the following example, the value is not specified, so the default value of `seq2` is used.
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    INSERT into t values();
+    ```
+
+    ```
+    Query OK, 1 row affected (0.00 sec)
+    ```
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT * from t;
+    ```
+
+    ```
+    +------+
+    | a    |
+    +------+
+    |    9 |
+    +------+
+    1 row in set (0.00 sec)
+    ```
+
++ In the following example, the value is not specified and the default value of `seq2` is not within the range defined in the above example (`CREATE SEQUENCE seq2 start 3 increment 2 minvalue 1 maxvalue 10 cache 3;`), so an error is returned.
```

Comment:
```suggestion
+ In the following example, the value is not specified, so the default value of `seq2` is used. But the next value of `seq2` is not within the range defined in the above example (`CREATE SEQUENCE seq2 start 3 increment 2 minvalue 1 maxvalue 10 cache 3;`), so an error is returned.
```
Rewrite according to the latest change in https://github.com/pingcap/docs-cn/pull/2567/files